src/main/java/gov/cabinetofice/gapuserservice/web/UserController.java

```diff
@@ -29,8 +30,18 @@ public class UserController {
     private final OneLoginUserService oneLoginUserService;
     private final DepartmentService departmentService;
     private final RoleService roleService;
+    private final CustomJwtServiceImpl jwtService;
     private final SecretAuthService secretAuthService;
 
+    @GetMapping("/userFromJwt")
+    public ResponseEntity<UserDto> getUserFromJwt(HttpServletRequest httpRequest) {
+        if (!roleService.isSuperAdmin(httpRequest)) {
+            throw new ForbiddenException();
```

Comment:
Can write unit test to cover this scenario

Reply:
Added a few to cover this controller

Reply:
![Screenshot 2023-09-12 at 09 52 41](https://github.com/cabinetoffice/gap-user-service/assets/135221918/b9138026-ce46-46ca-ac51-7c75f1d6eb1f)

No test to cover what happens when user is empty?

Reply:
added that